app/uk/gov/hmrc/cardpaymentfrontend/views/Views.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cardpaymentfrontend.views
+
+import uk.gov.hmrc.cardpaymentfrontend.views.html.EmailAddressPage
+
+import javax.inject.Inject
+
```

Comment:
This is something I've used/seen in other projects as a simple(er?) way to access the views from a common place, instead of injecting every view we need to each controller, instead you just use this to access the views.

Reply:
I'm doubtful about this. In pay-frontend there's a collection of forms all held together despite the fact that they have nothing in common beyond being Forms. Someone coming to this as a new codebase will have to go through the extra step of accessing the extra Views construct instead of being able to see exactly which views are used here by looking at what is injected. Form that POV this is more of a cognitive load for newcomers and I'm not clear what the advantage is.

Reply:
I'll remove this then